tpm2/test/tpm2_test.go

```diff
@@ -575,6 +575,144 @@ func TestCertify(t *testing.T) {
 	})
 }
 
+func TestCertifyEx(t *testing.T) {
+	rw := openTPM(t)
+	defer rw.Close()
+
+	params := Public{
+		Type:       AlgRSA,
+		NameAlg:    AlgSHA256,
+		Attributes: FlagSignerDefault,
+		RSAParameters: &RSAParams{
+			Sign: &SigScheme{
+				Alg:  AlgRSASSA,
+				Hash: AlgSHA256,
+			},
+			KeyBits: 2048,
+		},
+	}
+	signerHandle, signerPub, err := CreatePrimary(rw, HandleOwner, pcrSelection7, emptyPassword, defaultPassword, params)
+	if err != nil {
+		t.Fatalf("CreatePrimary(signer) failed: %s", err)
+	}
+	defer FlushContext(rw, signerHandle)
+
+	subjectHandle, subjectPub, err := CreatePrimary(rw, HandlePlatform, pcrSelection7, emptyPassword, defaultPassword, params)
+	if err != nil {
+		t.Fatalf("CreatePrimary(subject) failed: %s", err)
+	}
+	defer FlushContext(rw, subjectHandle)
+
+	attest, sig, err := CertifyEx(rw, defaultPassword, defaultPassword, subjectHandle, signerHandle, nil, SigScheme{Alg: AlgRSASSA, Hash: AlgSHA256, Count: 0})
```

Comment:
Nit: The `Count` member of SigScheme is not used in this case and not relevant to this call, and omitting it (letting it default to the same value, 0) seems more readable.